synapse/lib/stormlib/auth.py

```diff
@@ -1579,6 +1579,181 @@ async def getPermDef(self, perm):
         perm = await s_stormtypes.toprim(perm)
         return self.runt.snap.core.getPermDef(perm)
 
+@s_stormtypes.registry.registerType
+class StormUserVarsDict(s_stormtypes.Prim):
+    '''
+    A Storm Primitive that maps the HiveDict interface to a user vars dictionary.
```

Comment:
Is this comment about HiveDict still valid?

Reply:
Yeah it is using the older HiveDict convention of using `get` rather than the newer deref style behavior that the `UserVars` prim uses.

Reply:
This and the corresponding type for UserProfile will be removed in 3.0 where we will switch to just using the UserProfile prim